src/pages/index.tsx

```diff
@@ -42,6 +43,9 @@ const Home: NextPage = () => {
       className="flex flex-col w-screen h-screen text-white"
       id="masked-overflow"
     >
+      <div className="absolute flex flex-row w-full lg:justify-end justify-center lg:p-10 p-4 top-0 z-10">
```

Comment:
i think it looks better if it's right aligned on md viewports too. But i leave that decision up to you